nncf/torch/quantization/algo.py

```diff
@@ -679,7 +720,7 @@ def _get_quantizer_setup(self, target_model: NNCFNetwork) -> SingleConfigQuantiz
                                                                   self._precision_init_params,
                                                                   self._range_init_params,
                                                                   self._debug_interface)
-        single_config_quantizer_setup = setup_generator.generate_setup()
+        single_config_quantizer_setup = setup_generator.generate_setup(self._overflow_fix, self._use_logarithm_scale_per_group)
```

Comment:
It would be more clear, if setup generator knew all required parameters on initialization. Instead of passing on `generate_setup` method. I doubt an instance of the generator can be used multiple times with different parameters.